submission_rules.adoc

```diff
@@ -267,6 +271,22 @@ A submission must contain the following:
 *   Result logs for each system-implementation tested
 
 
+The submission must be submitted by the submission deadline either as a pull request to the submission repository or using the <<id_submission_privacy,private submission process>>. 
+
+#### Qualified Exceptions [[id_qualified_exceptions]]
+
+In the case where last-minute unforeseen events prevent a submission from being submitted to the submission repository or sent to the results chair, a Late Submitter is allowed to make a submission after the deadline following the procedure outlined below. This procedure is meant for extreme circumstances beyond the submitter's control. No submitter may make a Late Submission in two consecutive rounds unless by unanimous approval from the other submitters.
+
+
+
+*   Prior to the submission deadline, the Late Submitter must provide a hash of a tarball containing the submission as Proof of Work as well as an explanation of the reason for the delay via email to the other submitters as well as the results chair, so that the other parties can anticipate a late submission. The submission tarball must contain <<id_submission_content, all required content>>. 
+*   Once notified of a late submission, the chair will wait until all late submissions are resolved before releasing any of the <<id_submission_privacy,private submission results>>.
+*   The submitter then has 24 hours after the submission deadline to make the submission tarball available to the results chair and other submitters. The tarball must match the hash. If the hash has been verified and the majorityof other submitters do not object to the provided explanation, the results chair may accept the submission and make it and all other submissions available.  Otherwise the review committee must decide whether the Late Submission meets the criteria for acceptance, following the procedure described <<id_agenda_and_decisions,here>>.
```

Comment:
```suggestion
*   The submitter then has 24 hours after the submission deadline to make the submission tarball available to the results chair and other submitters. The tarball must match the hash. If the hash has been verified and the majority of other submitters do not object to the provided explanation, the results chair may accept the submission and make it and all other submissions available.  Otherwise the review committee must decide whether the Late Submission meets the criteria for acceptance, following the procedure described <<id_agenda_and_decisions,here>>.
```